src/schema/schema.ts

```diff
@@ -447,7 +448,16 @@ export function createZimbraSchema(
 				modifyWhiteBlackList: (_, { whiteBlackList }) =>
 					client.modifyWhiteBlackList(whiteBlackList as WhiteBlackListInput),
 				createTag: (_, { tag }) => client.createTag(tag as CreateTagInput),
-				tagAction: (_, { action }) => client.action(ActionType.tag, action)
+				tagAction: (_, { action }) => client.action(ActionType.tag, action),
+				suspendDeviceSync: (_, { deviceId }) => {
```

Comment:
as far as I know all queries/mutations invoked from user-sessions-management zimlet is directly hitting graphql instead of going from this layer

Reply:
I talked with Rupali and confirm the same by checking zm-mailbox/soap but there is no @graphql annotation at all.
This has no support from back-end. we need this layer.